bluesky/plan_patterns.py

```diff
@@ -29,6 +30,8 @@ def spiral(x_motor, y_motor, x_start, y_start, x_range, y_range, dr, nth):
         Delta radius
     nth : float
         Number of theta steps
+    tilt : float, optional
+        Tilt angle in degrees, default 0.0
```

Comment:
Standard practice in scipy is to make the interfaces radians, not degrees. Is that acceptable?